### PR TITLE
OBSDOCS-406 - Logging 5.7.4 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OBSDOCS-406

Link to docs preview: https://62992--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_7/logging-5-7-release-notes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by libander** [comment]
@anpingli / @kabirbhartiRH / @periklis / @jcantrill

**Review by jcantrill — APPROVED** [review]
/lgtm

**Comment by kabirbhartiRH** [comment]
Couple of bugs having 'Release Note Type' as Bug Fix but not in docs
https://issues.redhat.com/browse/LOG-4015
https://issues.redhat.com/browse/LOG-4372

Otherwise looks good

File: modules/logging-rn-5.7.4.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[id="openshift-logging-5-7-4-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, when forwarding logs to CloudWatch, a `namespaceUUID` value was not appended to the `logGroupName` field. With this update, the `namespaceUUID` value is included, so a `logGroupName` in CloudWatch appears as `logGroupName: vectorcw.b443fb9e-bd4c-4b6a-b9d3-c0097f9ed286`. (link:https://issues.redhat.com/browse/LOG-2701[LOG-2701])
+
+* Before this update, when forwarding logs over HTTP to an off-cluster destination, the Vector collector was unable to authenticate to the cluster-wide HTTP proxy even though correct credentials were provided in the proxy URL. With this update, the Vector log collector can now authenticate to the cluster-wide HTTP proxy. (link:https://issues.redhat.com/browse/LOG-3381[LOG-3381])
+
+* Before this update, the Operator would fail if the Fluentd collector was configured with Splunk as an output, due to this configuration being unsupported. With this update, configuration validation will reject unsupported outputs, preventing such issues from occurring. (link:https://issues.redhat.com/browse/LOG-4237[LOG-4237])
```
Comment (mburke5678) [inline modules/logging-rn-5.7.4.adoc:15]: s/will reject/rejects
s/such/this (_such_ seems vague to me)

File: modules/logging-rn-5.7.4.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+* Before this update, the Vector collector occasionally panicked with the following error message in its log:
+`thread 'vector-worker' panicked at 'all branches are disabled and there is no else branch', src/kubernetes/reflector.rs:26:9`. With this update, the error has been resolved. (link:https://issues.redhat.com/browse/LOG-4275[LOG-4275])
+
+* Before this update, an issue in the Loki Operator caused the `alert-manager` configuration for the application tenant to disappear if the Operator was configured with additional options for that tenant. With this update, the generated Loki configuration now contains both the custom and the auto-generated configuration. (link:https://issues.redhat.com/browse/LOG-4361[LOG-4361])
+
+* Before this update, When multiple roles were used to authenticate via STS with AWS Cloudwatch forwarding, a recent update caused the credentials to be non-unique. With this update, multiple combinations of STS roles and static credentials can once again be used to authenticate with AWS Cloudwatch. (link:https://issues.redhat.com/browse/LOG-4368[LOG-4368])
```
Comment (mburke5678) [inline modules/logging-rn-5.7.4.adoc:24]: s/When/when

**Comment by mburke5678** [comment]
@libander A few nits. Otherwise LGTM

**Pull request merged**